testing/php7-apcu/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Contributor: Valery Kartel <valery.kartel@gmail.com>
+# Maintainer: Carlo Landmeter <clandmeter@gmail.com>
+pkgname=php7-apcu
+_pkgreal=apcu
```

Comment:
Hm, you’re right, and not even php* packages, but also a perl* packages. Actually there are only slightly more packages that uses `_pkgname`.

I’ll discuss it on IRC later. For now, let’s keep `_pkgreal`.